client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -40,17 +41,30 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 		window.scrollTo( 0, 0 );
 	}, [ location ] );
 
-	flow.useAssertConditions?.();
+	const assertCondition = flow.useAssertConditions?.() ?? { state: AssertConditionState.SUCCESS };
+
+	const renderStep = ( path: StepPath ) => {
+		switch ( assertCondition.state ) {
+			case AssertConditionState.CHECKING:
+				/* eslint-disable wpcalypso/jsx-classname-namespace */
+				return <WordPressLogo size={ 72 } className="wpcom-site__logo" />;
+			/* eslint-enable wpcalypso/jsx-classname-namespace */
+			case AssertConditionState.FAILURE:
+				throw new Error( assertCondition.message ?? 'Error in Stepper pre-conditions' );
```

Comment:
Not a problem, but in case the message is missing I wouldn't tell the user about a "Stepper" problem :) I think something more generic would be better!